server/ruby/README.md

```diff
@@ -0,0 +1,64 @@
+# Stripe Payments Demo - Ruby Server
+
+This demo uses a simple [Sinatra](http://sinatrarb.com/) application as the server.
+
+## Payments Integration
+
+- [`app.rb`](app.rb) contains the routes that interface with Stripe to create charges and receive webhook events.
+- [`setup.rb`](setup.rb) a simple setup script to make some fake Products and SKUs for our Stripe store.
+- [`inventory.rb`](inventory.rb) a minimal wrapper over the Stripe Python SDK that handles creating/fetching products and caluclating payment amounts from SKUs. You can override this class with your own product and order management system code.
+
+## Requirements
+
+You’ll new the following:
+
+- [Ruby 2.X](https://www.ruby-lang.org/en/downloads/)
+- Modern browser that supports ES6 (Chrome to see the Payment Request, and Safari to see Apple Pay).
+- Stripe account to accept payments ([sign up](https://dashboard.stripe.com/register) for free!)
+
+## Getting Started
+
+Before getting started check that you have ruby installed
+
+```
+ruby --version
+```
+
+Copy the example environment variables file `.env.example` from the root of the repo into your own environment file called `.env`:
+
+```
+cp .env.example .env
+```
+
+User `bundler` to install the required gems
+
```

Comment:
maybe add that one should navigate to `server/ruby` before running these commands? Should be clear, but still.